README.Rmd

```diff
@@ -18,55 +17,87 @@ knitr::opts_chunk$set(
 
-{{ packagename }} provides functions to ....
+<!-- badges: end -->
 
-<!-- This sentence is optional and can be removed -->
-{{ packagename }} is developed at the [CENTER|similar](url) at the [UNIVERSITY|similar](url) as part of the [Epiverse-TRACE program](https://data.org/initiatives/epiverse/).
+**datatagr** provides functions to label and validate data of any kind. datatagr is an abstraction from [**linelist**](https://github.com/epiverse-trace/linelist), which applies these principles to epidemiological linelist data. The original proposal for this package can be found on [the Discussion board](https://github.com/orgs/epiverse-trace/discussions/221).
 
 ## Installation
 
-You can install the development version of {{ packagename }} from
+You can install the development version of datatagr from
 [GitHub](https://github.com/) with:
 
-``` r
+```r
 # install.packages("pak")
-pak::pak("{{ gh_repo }}")
+pak::pak("epiverse-trace/datatagr")
 ```
 
-## Example
+## Getting started
 
-These examples illustrate some of the current functionalities
+```r
+library(datatagr)
 
+# Create a datatagr object
+x <- make_datatagr(cars, speed = 'Miles per hour', dist = 'Distance in miles')
+
+# Validate the data are of a specific type
+validate_datatagr(x, 
+  speed = 'numeric',        # speed should be numeric
+  # type() is a helper function of related classes
+  dist = type('numeric')    # dist should be numeric, integer
+)
+```
 
 ## Development
 
 ### Lifecycle
 
-This package is currently a *concept*, as defined by the [RECON software
+This package is currently *experimental*, as defined by the [RECON software
 lifecycle](https://www.reconverse.org/lifecycle.html). This means that essential
 features and mechanisms are still being developed, and the package is not ready
 for use outside of the development team.
 
 ### Contributions
 
-Contributions are welcome via [pull requests](https://github.com/{{ gh_repo }}/pulls).
+Contributions are welcome via [pull requests](https://github.com/epiverse-trace/datatagr/pulls). Anything bigger than a typo fix or a small documentation update should be discussed in an issue first. If you want to report a bug or suggest an enhancement, please open an issue. 😊 See also [the general Epiverse TRACE contribution document](https://github.com/epiverse-trace/.github/blob/main/CONTRIBUTING.md).
+
+<details>
+  <summary>Common issues</summary>
+
+To make it easier for us to evaluate your contribution, please run the following commands before submitting a pull request to ensure your code is consistent with the rest of the package:
+
+```r
+styler::style_pkg()
+spelling::update_wordlist(pkg = ".", vignettes = TRUE)
+devtools::document()
+
+lintr::lint_package()
+
+devtools::test()
+devtools::check()
+```
+
+This will reduce the time it takes for us to review your contribution. Thank you! 😊
+
+</details>
+
 
 ### Related projects
```

Comment:
I believe `labelled` should be listed here as well. It's a good opportunity to highlight the differences with this package.